cmd/drone-buildx-ecr/main.go

```diff
@@ -40,6 +40,7 @@ func main() {
 		assumeRole       = getenv("PLUGIN_ASSUME_ROLE")
 		externalId       = getenv("PLUGIN_EXTERNAL_ID")
 		scanOnPush       = parseBoolOrDefault(false, getenv("PLUGIN_SCAN_ON_PUSH"))
+		idToken          = os.Getenv("PLUGIN_OIDC_TOKEN_ID")
```

Comment:
Why isn't this `getenv("PLUGIN_OIDC_TOKEN_ID")` like the lines above?

Reply:
was testing it locally that way. updated it to match with the existing standards by calling `getenv` func that calls the `os.Getenv`